packages/client/hmi-client/src/components/widgets/tera-chart-settings-panel.vue

```diff
@@ -93,6 +98,7 @@ const emit = defineEmits(['close', 'update-settings', 'delete-annotation', 'crea
 
 // Log scale
 const useLog = computed(() => props.activeSettings?.scale === 'log');
+const isHiddenInNode = computed(() => !!props.activeSettings?.hideInNode);
```

Comment:
Why not casting it as boolean here to avoid the casting in the prop `:model-value` line 18?
```suggestion
const isHiddenInNode = computed<boolean>(() => !!props.activeSettings?.hideInNode);
```

Reply:
The boolean casting is also there for `useLog` and `isSmallMultiples` so I was continuing the pattern; but I like this better I'll change the others too